samples/client/petstore/groovy/bin/main/org/openapitools/model/InlineObject.groovy

```diff
@@ -0,0 +1,13 @@
+package org.openapitools.model;
```

Comment:
Why are there changes to groovy and kotlin samples?

Is your change based on the recent/latest master of the official repo?